src/test/suite/fixtures/recorded/languages/rust/chuckArgTrap.yml

```diff
@@ -0,0 +1,32 @@
+languageId: rust
+command:
+  spokenForm: chuck arg trap
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: t}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: argumentOrParameter}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    struct S<'a, 'b: 'a, T>;
+  selections:
+    - anchor: {line: 1, character: 0}
+      active: {line: 1, character: 0}
+  marks:
+    default.t:
+      start: {line: 0, character: 21}
+      end: {line: 0, character: 22}
+finalState:
+  documentContents: |
+    struct S<'a, 'b: 'a, >;
```

Comment:
This one doesn't look right to me.  Shouldn't this one have removed the trailing comma?

Reply:
Yes, that’s strange.

Reply:
That's the `: 'a` thing? Agreed

Reply:
Yes. It’s rarely used so I forgot the official name.

Reply:
Okay, It’s actually just a trait bound for the tree sitter.

Reply:
Cool so "type" / "name" just works out of the box then?

Any idea why comma is sticking around?